Dockerfile

```diff
@@ -69,7 +70,9 @@ RUN glide up --skip-test \
     && go install github.com/lyft/protoc-gen-validate \
     && go install github.com/mwitkow/go-proto-validators/protoc-gen-govalidators \
     && go install github.com/pseudomuto/protoc-gen-doc/cmd/... \
-    && go install github.com/infobloxopen/protoc-gen-gorm \
+    && go install  \
+      -ldflags "-X github.com/infobloxopen/protoc-gen-gorm/plugin.ProtocGenGormVersion=$PGG_VERSION -X github.com/infobloxopen/protoc-gen-gorm/plugin.AtlasAppToolkitVersion=$AAT_VERSION" \
```

Comment:
is there any way to have this instead be set from the `go install` command? That seems like it would make more sense to me